<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDefinitionWizard</class>
 <widget class="QWizard" name="AddDefinitionWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>834</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wizard</string>
  </property>
  <widget class="QWizardPage" name="implementDefinitionWizardPage">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="definitionNameLabel">
        <property name="text">
         <string>Definition Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="definitionName">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="drawImplementationWizardPage">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="RenderArea" name="renderer" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Fill Colour</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="brushColourButton">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="penColourLabel">
          <property name="text">
           <string>Outline Colour:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="penColourButton">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="shapeLabel">
          <property name="text">
           <string>Shape: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="shapeComboBox">
          <property name="currentText">
           <string>Freeform</string>
          </property>
          <item>
           <property name="text">
            <string>Freeform</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Line</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Rectangle</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Triangle</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="penWidthLabel">
          <property name="text">
           <string>Pen Width:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="penWidthSpinBox">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RenderArea</class>
   <extends>QWidget</extends>
   <header>renderarea.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
